packages/dev/loaders/src/glTF/2.0/Extensions/KHR_animation_pointer.map.ts

```diff
@@ -0,0 +1,510 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { Animation } from "core/Animations/animation";
+import type { AnimationGroup } from "core/Animations/animationGroup";
+import { Quaternion, Vector3, Matrix } from "core/Maths/math.vector";
+import { Color3 } from "core/Maths/math.color";
+import type { IGLTF } from "../glTFLoaderInterfaces";
+import { Material } from "core/Materials/material";
+import type { IAnimatable } from "core/Animations/animatable.interface";
+import type { Mesh } from "core/Meshes/mesh";
+import type { Nullable } from "core/types";
+
+export type GetGltfNodeTargetFn = (source: IGLTF, indices: string) => any;
+export type GetValueFn = (target: any, source: Float32Array, offset: number, scale?: number) => any;
+export type GetStrideFn = (target: any) => number;
+
+export interface IAnimationPointerPropertyInfos {
+    type: number;
+    name: string;
+    get: GetValueFn;
+    isValid(target: any): boolean;
+    buildAnimations(target: any, fps: number, keys: any[], group: AnimationGroup, animationTargetOverride: Nullable<IAnimatable>, params?: any): void;
+}
+
+const _parseIntIndex = (str: string) => {
+    const targetIndex = parseInt(str);
+    if (isNaN(targetIndex)) {
+        return -1;
+    }
+    return targetIndex;
+};

```

Comment:
No need for `_` for non-exported local functions and you can use function syntax. Ditto elsewhere.
``````suggestion
function parseIntIndex(str: string): number {
    const targetIndex = parseInt(str);
    if (isNaN(targetIndex)) {
        return -1;
    }
    return targetIndex;
}
```